### PR TITLE
Update hooks-effect.md

### PR DESCRIPTION
**Pull request by EngincanV**

### CODE REVIEW

File: content/docs/hooks-effect.md [hunk 1]
```diff
@@ -468,22 +467,24 @@ useEffect(() => {
...
+>["Eslint-plugin-react-hooks"](https://www.npmjs.com/package/eslint-plugin-react-hooks#installation) paketimizin bir parçası olan [`exhaustive-deps`](https://github.com/facebook/react/issues/14920) kuralını kullanmanızı öneririz. Bağımlılıklar yanlış belirlendiğinde uyarı verir ve bir düzeltme önerir.
 
-## Next Steps {#next-steps}
+## İleriki Adımlar {#next-steps}
 
-Congratulations! This was a long page, but hopefully by the end most of your questions about effects were answered. You've learned both the State Hook and the Effect Hook, and there is a *lot* you can do with both of them combined. They cover most of the use cases for classes -- and where they don't, you might find the [additional Hooks](/docs/hooks-reference.html) helpful.
+Tebrikler! Bu uzun bir sayfaydı, ama umarım sonunda etkilerle ilgili sorularınızın çoğu yanıtlanmıştır. Hem State Hook'unu hem de Effect Hook'unu öğrendiniz ve ikisini birleştirerek yapabileceğiniz *çok fazla* şey var. Sınıflar için kullanım durumlarının çoğunu kapsar - ve olmadıkları yerlerde, [ek Hook'lar](/docs/hooks-reference.html) yararlı olabilir.
```
Comment (alioguzhan) [inline content/docs/hooks-effect.md:486]: `umarim` -> `umariz`

**Review by alioguzhan — CHANGES_REQUESTED** [review]
Thanks for the translation. Left a comment just for one place. Please check.

**Comment by EngincanV** [comment]
@alioguzhan required change has been made.

**Review by alioguzhan — APPROVED** [review]

**Comment by alioguzhan** [comment]
Thanks for the translation. Merging it...

**Pull request merged**